frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.tsx

```diff
@@ -232,6 +237,7 @@ function StartingJobModal(props: StartingJobModalProps) {
   const tracing = useSelector((state: OxalisState) => state.tracing);
   const activeUser = useSelector((state: OxalisState) => state.activeUser);
   const layers = chooseSegmentationLayer ? getSegmentationLayers(dataset) : getColorLayers(dataset);
+  const allLayer = getDataLayers(dataset);
```

Comment:
```suggestion
  const allLayers = getDataLayers(dataset);
```

usages need to be changed then, too.